### DISCUSSION TITLE
Have certain field part of the database for selection

### DISCUSSION THREAD

**Discussion by jnelson22** [sysadminsmedia/homebox#730] [Ideas]
### What is the problem you are trying to solve with this feature?

Allow for easily adding multiple items from the "Manufacturer" or "Purchased From". Allowing for data consistency across assets from the same manufacturer or purchased from 

### What is the solution you are proposing?

Allow those fields to have a dropdown that pulls from the database

### What alternatives have you considered?

No

### Contributions

- [X] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712
